netty/src/main/java/io/grpc/netty/NettyClientTransport.java

```diff
@@ -165,14 +166,20 @@ public ClientStream newStream(MethodDescriptor<?, ?> method, Metadata headers) {
   public Runnable start(Listener transportListener) {
     lifecycleManager = new ClientTransportLifecycleManager(
```

Comment:
`lifecycleManager` is used in `statusFromFailedFuture`, so it must remain a field. I left `keepaliveManager` because it is used by the test... although I agree that is questionable given the test didn't catch this bug.